docs/install.rst

```diff
@@ -25,10 +25,10 @@ on network speed and computing power.
 
 .. _configure_securedrop:
 
-Localization of the source and journalist interfaces
-----------------------------------------------------
+Localization of the User Interfaces
```

Comment:
We do not have **User Interfaces** anywhere else. Would it be a problem to use 

    *Source Interface* and *Journalist Interface* 

instead?

Reply:
:heavy_check_mark: